solidity/truffle-examples/url-requests/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Oraclize Examples using Truffle: Computation-Datasource - Url Requests",
```

Comment:
npm will complain but silently fail due to this invalid name. Same with pnpm (it throws errors in fact). https://www.npmjs.com/package/validate-npm-package-name

Reply:
TIL! 

Have renamed to the following: `"oraclize-examples-using-truffle__computation-datasource-url-requests"`

Suitable?